productivity.md

```diff
@@ -0,0 +1,75 @@
+- Some work is very difficult to plan by just looking at it. Make prototypes to help gain a better understanding and identify potential issues.
+- Ask: What are some non-obvious interactions that might cause problems?
+- In planning meetings, get everyone to speak, so everyone is able to share their unique perspective. There might be long periods of silence while folks are thinking, and that’s ok.
+
+### Handling grossly misestimated tickets
+
+- Work with the backlog owner to reduce the scope and update the DoD.
+- Create additional tickets to encompass the remaining work.
+- Working with the backlog owner, determine whether the additional tickets should be pulled into the current sprint or prioritized for an upcoming sprint.
+
+## Troubleshooting individual productivity
+
+### Consistently low velocity (typically junior)
+
+Low velocity is usually a symptom of either time management issues or taking tickets that are too difficult.
+
+- Limit work-in-progress. You should never have more than 3 tickets open at a time. Ideally you should only ever 1 in progress and 1-2 in PR. Focus on one thing at a time to reduce the need for context switching.
+- Use the guide for [prioritizing work](#prioritizing-work) to decide what you should be working on.
+- Take small, well-defined tickets. Avoid tickets 3-points or higher, as these usually contain large areas of uncertainty.
+- Before starting any ticket, make a plan for how you will accomplish the ticket. Present that plan to a more senior engineer and get feedback.
+  - For larger tasks, repeat this process daily, revising your plan as you go along.
+- If you struggle with rabbit-holing, set a repeating timer for 30 minutes. When the timer goes off, if you don’t feel like you’re making meaningful progress, reach out for help.
+  - Once you are consistently making progress, increase the timer interval until you feel confident that you can notice when you’re stuck and need help.
+
+As a manager of a team member struggling with low velocity, begin by asking open-ended questions to get the developer to reflect on why they’ve been struggling to deliver tickets.
+
+- How does the person feel about their tickets?
+  - Can they identify any trends in the problems they encounter with tickets?
+- How do they feel asking for help?
+  - What prevents them from noticing that they should ask for help?
+  - What prevents them from reaching out for help when they notice they’re stuck?
+
+### Unpredictable / highly variable velocity (typically mid-level)
+
+When a developer’s productivity fluctuates dramatically, this is often a symptom of taking on highly complex work relative to their level.
+
+- Learning complex problem-solving takes time; keep at it. Unfortunately, the long term solution to this problem is to just keep practicing until you gain mastery over new skills. Keep challenging yourself, just make sure you don’t slip into consistently low velocity.
```

Comment:
Agreed that this is a tough skill to learn, that said: we should be able to challenge ourselves and solve complex problems without observing variable velocity or consistently low sprints. 

If we're going to challenge ourselves, it's **more** important to do more thorough planning with the rest of the team to discuss the approach, identify the areas of uncertainty, and to create the correct set of tickets. 

The second bullet point here is the most important of the three.

Reply:
Do you think there is a way to clarify or better emphasize this point?

Reply:
My recommendation here is just to put your last bullet point at the top of the list.